### PR TITLE
added additional antenna selection descriptions

### PR DESCRIPTION
**Pull request by gijsio**
accompanying PR 198 on FW Dev

### CODE REVIEW

File: content/firmwareapi/pycom/network/bluetooth/_index.md [hunk 1]
```diff
@@ -67,24 +67,19 @@ bluetooth = Bluetooth()
...
 * `privacy` Enables or Disables local privacy settings so address will be random or public.
 * `secure_connections` Enables or Disables Secure Connections and MITM Protection.
 * `mtu` Maximum Transmission Unit (MTU) is the maximum length of an ATT packet. Value must be between `23` and `200`.
 
 
-  With our development boards it defaults to using the internal antenna, but in the case of an OEM module, the antenna pin (`P12`) is not used, so it's free to be used for other things.	With our development boards it defaults to using the internal antenna, but in the case of an OEM module, the antenna pin (`P12`) is not used, so it's free to be used for other things.
-Initialises and enables the Bluetooth radio in BLE mode.
+    With our development boards it defaults to using the internal antenna, but in the case of an OEM module, the antenna pin (`P12`) is not used, so it's free to be used for other things.	With our development boards it defaults to using the internal antenna, but in the case of an OEM module, the antenna pin (`P12`) is not used, so it's free to be used for other things.
```
Comment (peter-pycom) [inline content/firmwareapi/pycom/network/bluetooth/_index.md]: can we fix the duplicated text as well while we're here?

File: content/firmwareapi/pycom/network/bluetooth/_index.md [hunk 2]
```diff
@@ -67,24 +67,19 @@ bluetooth = Bluetooth()
...
+    Initialises and enables the Bluetooth radio in BLE mode.
 
-To use an external antenna, set `P12 as output pin.`
-
-```python
-
-Pin('P12', mode=Pin.OUT)(True)
-```
```
Comment (peter-pycom) [inline content/firmwareapi/pycom/network/bluetooth/_index.md:82]: why are we removing this? actually, I not sure I fully get it .... It's like a decision tree right?

- INT
- EXT
- MAN
  - do xyz for int, abc, for ext, how exactly? what is it by default?

File: content/firmwareapi/pycom/network/wlan.md [hunk 3]
```diff
@@ -175,7 +176,11 @@ Possible channels are in the range of 1-14, depending on your country settings.
 ### wlan.antenna([antenna])
 
 Get or set the antenna type (external or internal). Value can be:
+  * `WLAN.INT_ANT`: The on-board antenna
+  * `WLAN.EXT_ANT`: The U.FL connector (external antenna)
+  * `WLAN.MAN_ANT`: Manually select the state of the antenna switch on `P12`. By default, this will select the on-board antenna
 
+    With our development boards it defaults to using the internal antenna, but in the case of an OEM module, the antenna pin (`P12`) is not used, so it's free to be used for other things.	With our development boards it defaults to using the internal antenna, but in the case of an OEM module, the antenna pin (`P12`) is not used, so it's free to be used for other things.
```
Comment (peter-pycom) [inline content/firmwareapi/pycom/network/wlan.md]: duplicated

**Review by peter-pycom — CHANGES_REQUESTED** [review]

File: content/firmwareapi/pycom/network/bluetooth/_index.md (hunk 2 above)
Comment (gijsio) [reply content/firmwareapi/pycom/network/bluetooth/_index.md:82]: I think the small examples in between, like this, make the API pages cluttered.

**Review by peter-pycom — APPROVED** [review]

**Pull request merged**